Overview.src.html

```diff
@@ -1589,6 +1589,11 @@ <h3 id="should-response-to-request-be-blocked-due-to-mime-type?"><dfn title="sho
  <li><p>Return <var>reader</var>.
 </ol>
 
+<p>To <dfn title=concept-read-chunk-from-ReadableStream>read a chunk</dfn> from a
+<code title=concept-ReadableStream>ReadableStream</code> object with <var>reader</var>, return the
+result of calling
+<code data-anolis-spec=streams>ReadFromReadableStreamReader</code>(<var>reader</var>).
```

Comment:
Actually, maybe we should not use `<code>` here? I don't think we do that in HTML.